Gifski/AppDelegate.swift

```diff
@@ -36,13 +34,11 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
 		mainWindowController.showWindow(self)
 
-		hasFinishedLaunching = true
 		NSApp.isAutomaticCustomizeTouchBarMenuItemEnabled = true
 		NSApp.servicesProvider = self
 
-		if urlToConvertOnLaunch != nil {
-			mainWindowController.convert(urlToConvertOnLaunch)
-		}
+		// Set launch completions option if the notification center could not be set up already
```

Comment:
Why is this needed? Or rather, why does it matter if the notification is posted slightly later than in this function?